packages/react/src/components/Panel/index.tsx

```diff
@@ -1,33 +1,41 @@
-import React from 'react';
+import React, { HTMLAttributes, ReactElement, ReactNode } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import IconButton from '../IconButton';
 
-const Panel = ({
-  className,
-  title = '',
-  actions,
-  children
-}: {
+interface PanelProps extends HTMLAttributes<HTMLElement> {
+  children: ReactNode;
+  heading:
+    | ReactElement<any>
+    | {
+        text: ReactElement<any>;
+        level: number | undefined;
+      };
   className?: string;
-  title?: string;
-  actions?: typeof IconButton[];
-  children: React.ReactNode;
-}) => {
+}
+
+const Panel = ({ children, className, heading, ...other }: PanelProps) => {
+  const Heading = `h${
+    typeof heading === 'object' && 'level' in heading && !!heading.level
+      ? heading.level
+      : 2
+  }` as 'h1';
+
   return (
-    <div className={classNames('Panel', className)}>
-      <div className="Panel__Header">
-        {title && <div className="Panel__Header-title">{title}</div>}
-        {actions && <div className="Panel__Header-actions">{actions}</div>}
-      </div>
-      <div className="Panel__Content">{children}</div>
-    </div>
+    <section className={classNames('Panel', className)} {...other}>
```

Comment:
This seems like it should be a section to me. However, I'm open to being corrected.

Reply:
I agree with you here

Reply:
you should `aria-labelledby` this `<section>` by pointing it at the heading (accessibility best practice)

Reply:
@dylanb Thanks. I looked elsewhere in our apps where we were using `<section>`, and it didn't seem like we were using `aria-labelledby`, which seemed odd to me. I will go ahead and add it here.